app.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+//An array for use when creating the sales projections per hour
+var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
```

Comment:
Keeping global variables up at the top like this is super smart. So is commenting it like you did!